DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -58,7 +61,13 @@ internal DiscordApiClient(BaseDiscordClient client)
             this.Rest = new RestClient(proxy, timeout, useRelativeRateLimit, logger);
         }
 
-        private static string BuildQueryString(IDictionary<string, string> values, bool post = false)
+        /// <summary>
+        /// Builds a query string.
+        /// </summary>
+        /// <param name="values">Values to use.</param>
+        /// <param name="post"></param>
```

Comment:
Missing documentation for post

Reply:
Oh yes forgot about that. Don't really know what to put in there. Inputs would be appreciated.

Reply:
Post seems to append a query modifier onto the route. Not sure why it's called post, though.